packages/cli-vector/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:20-slim as builder
```

Comment:
This is just temporary file that I copy here as reference, we will move all the installs and ENV into package/cli Dockerfile later. Then test the vector cli in the cli container